carbon-design-system/carbon-for-ibm-dotcom
# feat(video-player-lightbox): Media viewer onClick enhancement + video image hover state (8% darken) treatment

### Related Ticket(s)

DDS Ticket: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/5245
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-1735

### Description
- This PR adds the ability to trigger the video player via the lightbox onClick instead of triggering the playing of the video inline.
- This is a requirement that came in through the AEM Solution template.

![Screen Shot 2021-03-25 at 4 28 27 PM](https://user-images.githubusercontent.com/1815714/112539278-2e310d00-8d87-11eb-8e7b-c0e45dcc8af3.png)


### Changelog

**New Prop**

- `playing-mode` - The video player's mode showing Inline or Lightbox.
- Implement the `8% darken` hover state treatment based on the design requirement requested by @shixiedesign.

**Removed**

- N/A

**Testing Instructions**
- Visit the `with Lightbox Media Viewer` - `?path=/story/components-video-player--with-lightbox-media-viewer`
- Ensure that the lightbox shows up after clicking on the video player thumbnail.
